contracts/IErrors.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.9;
+
+interface IErrors {
+    // auth
+    error NotOwner();
+    error NotBackend();
+    error NotNode();
+    error NotSFC();
+    error NotDriver();
+    error NotDriverAuth();
+    error NotContract();
+    error NotAuthorized();
+
+    // initialization
+    error ContractInitialized();
+
+    // reentrancy
+    error ReentrantCall();
```

Comment:
Please rename to `ReentrancyGuardReentrantCall` for consistency with OpenZepelin's ReentrancyGuard.
Please keep this defined in the ReentrancyGuard, there is no need to move it out of it.